tests/ssg_test_suite/test_env.py

```diff
@@ -120,6 +119,12 @@ class VMTestEnv(TestEnv):
 
     def __init__(self, mode, hypervisor, domain_name):
         super(VMTestEnv, self).__init__(mode)
+
+        try:
+            import libvirt
+        except ImportError:
+            raise RuntimeError("Can't import libvirt module, libvirt backend will therefore not work.")
```

Comment:
It would be nice to specify in README that libvirt is now only an optional dependency

Reply:
I have some additions regarding required packages that I can provide soon.

Reply:
This is out of scope of this PR. We don't have the `libvirt` dependency documented here at all.